XU-TechnicalPaper.md

```diff
@@ -25,163 +25,176 @@
 
 1.2. Project Intro
 ------------------
-This specification describes and summarizes commonly agreed technical requirements for Exchange Union and thus should be referred to as “the bible” ;). Exchange Union’s implementation is challenging and can only be achieved with support of a greater developer community. The decision was made to open-source all documentation and the entire code base and to prioritize building a developer community, while also engaging full-time open-source developers to contribute to the development effort. Strategic partnerships with relevant companies will be formed. The business model for funding development is based entirely on Exchange Union's XUC token, which is already in circulation and doesn’t require an ICO.
+The Exchange Union project is powered by [XUD](https://github.com/ExchangeUnion/xud/), short for **Exchange Union Daemon**, the  open source node software which is operated by exchanges (or traders) and comprises the network. This document describes the inner workings of XUD from a product & technical perspective and summarizes technical requirements. For an overview of idea, use case & benefits read the [White Paper](https://www.exchangeunion.com/static/files/ExchangeUnion-WhitePaper_en.pdf), for hands-on guides check our [Wiki](https://github.com/ExchangeUnion/xud/wiki) and for code-level documentation our [TypeDoc](https://exchangeunion.github.io/xud-typedoc/). Chat with us on [Gitter](https://gitter.im/exchangeunion/Lobby) and follow major event & development milestones on our [blog](https://blog.exchangeunion.com/).
+
+XUD's code base and documentation is available under the [AGPL-3.0](https://github.com/ExchangeUnion/xud/blob/master/LICENSE) open-source license. The objective is to build a open-source developer community, while additionally employing full-time developers under the Exchange Union brand. Development is funded through an initial private sale of 2% of Exchange Union's XUC token, which is since in circulation. No ICO or further fundraising is planned.
 
 
 ## 2. Product Description
-Exchange Union can be described as a *decentralized meta-exchange*, because it targets today's centralized exchanges as "users". This direction was chosen because the majority of users use and will use centralized exchanges for convenience and thus the liquidity is with centralized exchanges. Furthermore, executing and maintaining Exchange Union’s technology, XUD, short for **Exchange Union Daemon** is not expected to be suitable for personal or widespread usage in the near future. However, it is planned to further develop and simplify the technology and make it suitable for individual usage in the long-term. 
+From a business perspective, Exchange Union can be described as a *decentralized meta-exchange*: target users are existing centralized exchanges, in the following 'exchanges' for short.
```

Comment:
Instead of "in the following" I'd prefer "hereafter" which is a more commonly used expression I believe.

Reply:
Fixed